src/engines/html5/capabilities/html5-autoplay.js

```diff
@@ -56,6 +64,9 @@ export default class Html5AutoPlayCapability implements ICapability {
    * @private
    */
   static _getPlayPromise(): Promise<*> {
+    testVideoElement.addEventListener(Html5EventType.ERROR, () => {
+      return Promise.reject();
```

Comment:
This promise reject is not part of the `_forcePromiseReturnValue` promise you create there so 1. it won't solve the promise you need and 2. as you return this from an event listener this will just emit a global promise rejection not tied to anything.
Need to listen to error event in the `_forcePromiseReturnValue` and use the reject you got from the promise you created there.
Some browsers have different behaviours, not all will have error events I think so need to also listen to pause and abort possibly(I think FF had this issue)